src/app/public/modules/tabs/tab-dropdown.component.ts

```diff
@@ -42,9 +42,7 @@ export class SkyTabDropdownComponent {
   }
 
   public selectTab(tab: SkyTabComponent) {
-    if (!tab.disabled) {
```

Comment:
This condition can never be met in a unit test because the parent components detect the disabled state and abort any user actions before it gets to this method.